datalad/ui/progressbars.py

```diff
@@ -66,7 +85,85 @@ def __init__(self, label='', fill_text=None, total=None, unit='B', out=sys.stdou
         super(SilentProgressBar, self).__init__(total=total)
 
 
-progressbars = {'silent':  SilentProgressBar}
+class LogProgressBar(ProgressBarBase):
+    """A progress bar which logs upon completion of the item
+    """
+    def __init__(self, *args, **kwargs):
+        super(LogProgressBar, self).__init__(*args, **kwargs)
+        # I think we never generate progress bars unless we are the beginning
+        # of reporting something lengthy.  .start is not always invoked
```

Comment:
s/are/are at/, though I'm missing the connection between .start being invoked and recording the start time here.